libs/database/src/lib/entities/delayed-job.entity.ts

```diff
@@ -26,13 +26,21 @@ export class DelayedJob extends Model<DelayedJob> {
 
   @AllowNull
   @Column(INTEGER)
-  total_content: number | null;
+  totalContent: number | null;
 
   @AllowNull
   @Column(INTEGER)
-  processed_content: number | null;
+  processedContent: number | null;
 
   @AllowNull
   @Column(STRING)
-  proccess_message: string | null
+  progressMessage: string | null
+
+  @AllowNull
+  @Column(BIGINT)
```

Comment:
This needs to match what laravel uses, which is `BIGINT.UNSIGNED`

Reply:
changed